clusterloader2/pkg/config/cluster.go

```diff
@@ -42,6 +42,7 @@ type ClusterConfig struct {
 	MasterName                 string
 	KubemarkRootKubeConfigPath string
 	DeleteStaleNamespaces      bool
+	DeleteAutomanagedNamespaces bool
```

Comment:
I think gofmt will fail with it - please run gofmt.

Reply:
Checked the format